pkg/flow/grpc-namespace-variables.go

```diff
@@ -8,507 +8,190 @@ import (
+		Key:       req.GetKey(),
+	})
 	if err != nil {
 		return err
 	}
-
-	rdr := bytes.NewReader(vdata.Data)
-
-	for {
-		resp := new(grpc.NamespaceVariableResponse)
-
-		resp.Namespace = cached.Namespace.Name
-		resp.Key = vref.Name
-		resp.CreatedAt = timestamppb.New(vdata.CreatedAt)
-		resp.UpdatedAt = timestamppb.New(vdata.UpdatedAt)
-		resp.Checksum = vdata.Hash
-		resp.TotalSize = int64(vdata.Size)
-		resp.MimeType = vdata.MimeType
-
-		buf := new(bytes.Buffer)
-		k, err := io.CopyN(buf, rdr, parcelSize)
-		if err != nil {
-			if errors.Is(err, io.EOF) {
-				err = nil
-			}
-
-			if err == nil && k == 0 {
-				if resp.TotalSize == 0 {
-					resp.Data = buf.Bytes()
-					err = srv.Send(resp)
-					if err != nil {
-						return err
-					}
-				}
-				return nil
-			}
-
-			if err != nil {
-				return err
-			}
-		}
-
-		resp.Data = buf.Bytes()
-
-		err = srv.Send(resp)
-		if err != nil {
-			return err
-		}
+	err = srv.Send(resp)
+	if err != nil {
+		return err
 	}
-}
 
-var variablesOrderings = []*orderingInfo{
-	{
-		db:           entvar.FieldName,
-		req:          "UPDATED",
-		defaultOrder: ent.Asc,
-	},
-}
-
-var variablesFilters = map[*filteringInfo]func(query *ent.VarRefQuery, v string) (*ent.VarRefQuery, error){
-	{
-		field: util.PaginationKeyName,
-		ftype: "CONTAINS",
-	}: func(query *ent.VarRefQuery, v string) (*ent.VarRefQuery, error) {
-		return query.Where(entvar.NameContains(v)), nil
-	},
+	return nil
 }
 
 func (flow *flow) NamespaceVariables(ctx context.Context, req *grpc.NamespaceVariablesRequest) (*grpc.NamespaceVariablesResponse, error) {
 	flow.sugar.Debugf("Handling gRPC request: %s", this())
 
-	cached := new(database.CacheData)
-
-	err := flow.database.NamespaceByName(ctx, cached, req.GetNamespace())
+	ns, err := flow.edb.NamespaceByName(ctx, req.GetNamespace())
 	if err != nil {
 		return nil, err
 	}
-
-	clients := flow.edb.Clients(ctx)
-
-	query := clients.VarRef.Query().Where(entvar.HasNamespaceWith(entns.ID(cached.Namespace.ID)))
-
-	results, pi, err := paginate[*ent.VarRefQuery, *ent.VarRef](ctx, req.Pagination, query, variablesOrderings, variablesFilters)
+	_, store, _, rollback, err := flow.beginSqlTx(ctx)
 	if err != nil {
 		return nil, err
 	}
+	defer rollback()
 
-	resp := new(grpc.NamespaceVariablesResponse)
-	resp.Namespace = cached.Namespace.Name
-	resp.Variables = new(grpc.Variables)
-	resp.Variables.PageInfo = pi
-
-	err = bytedata.ConvertDataForOutput(results, &resp.Variables.Results)
+	list, err := store.RuntimeVariables().ListByNamespaceID(ctx, ns.ID)
 	if err != nil {
 		return nil, err
 	}
 
-	for i := range results {
-		vref := results[i]
-
-		vdata, err := vref.QueryVardata().Select(entvardata.FieldCreatedAt, entvardata.FieldHash, entvardata.FieldSize, entvardata.FieldUpdatedAt).Only(ctx)
-		if err != nil {
-			return nil, err
-		}
+	resp := new(grpc.NamespaceVariablesResponse)
+	resp.Namespace = ns.Name
+	resp.Variables = new(grpc.Variables)
+	resp.Variables.PageInfo = nil
```

Comment:
TODO?

Reply:
Not really, PageInfo should be null, the ui is not broken.